rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,501 @@
+will do the majority of project management and execution for the
+following deliverables. They will coordinate and collaborate with DCP
+partners ("contact person") to validate design correctness and minimize
+code changes and their impact.
+
+| Component/Deliverable                                 	| Responsible Team/Service 	| No earlier than 	| Contact Person	|
+|-------------------------------------------------------	|--------------------------	|-----------------	|---------------	|
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis Pipelines                                    	| Broad                    	| Q2 2019         	| ?             	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| TBD             	| S. Ehsan      	|
+| Data Analysis Service starfish Pipeline Tests         	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| Z. Perova             	|
+| [Ability to describe files that belong together][a]   	| ?                        	| ?               	| ?             	|
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| ?             	|
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| TBD             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| TBD             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| TBD             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| ?             	|
```

Comment:
```suggestion
| Image file download latency testing                   	| ?                        	| ?               	| B. Hannafious             	|

Reply:
@xbrianh if you're not the right contact for this, could you suggest someone else?